src/rpc/blockchain.cpp

```diff
@@ -777,22 +777,23 @@ static RPCHelpMan getmempoolentry()
 
 static RPCHelpMan getblockfrompeer()
 {
-    return RPCHelpMan{"getblockfrompeer",
-                "\nAttempt to fetch block from a given peer.\n"
-                "\nWe must have the header for this block, e.g. using submitheader.\n"
-                "\nReturns {} if a block-request was successfully scheduled\n",
-                {
-                    {"blockhash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The block hash"},
-                    {"nodeid", RPCArg::Type::NUM, RPCArg::Optional::NO, "The node ID (see getpeerinfo for node IDs)"},
-                },
-                RPCResult{RPCResult::Type::OBJ, "", "",
-                {
-                    {RPCResult::Type::STR, "warnings", "any warnings"}
-                }},
-                RPCExamples{
-                    HelpExampleCli("getblockfrompeer", "\"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09\" 0")
-                + HelpExampleRpc("getblockfrompeer", "\"00000000c937983704a73af28acdec37b049d214adbda81d7e2a3dd146f6ed09\" 0")
-                },
+    return RPCHelpMan{
+        "getblockfrompeer",
+        "\nAttempt to fetch block from a given peer.\n"
+        "\nWe must have the header for this block, e.g. using submitheader.\n"
+        "\nReturns {} if a block-request was successfully scheduled\n",
```

Comment:
```diff
-        "\nReturns {} if a block-request was successfully scheduled\n",
+        "\nReturns {} if a block-request was successfully scheduled.\n",
```

Reply:
The trailing newline isn't needed either. Whitespace will be normalized by the doc generator.

Reply:
Sorry, I never implemented this. Done in https://github.com/bitcoin/bitcoin/pull/23714